pyaedt/filtersolutions_core/attributes.py

```diff
@@ -0,0 +1,1528 @@
+    def upper_frequency(self) -> str:
+        """Filter upper corner frequency.
+        The default is `1.105 MHz`.
+
+        Returns
+        -------
+        str
+        """
+        upper_freq_string = self._dll_interface.get_string(self._dll.getUpperFrequency)
+        return upper_freq_string
+
+    @upper_frequency.setter
+    def upper_frequency(self, upper_freq_string):
+        self._dll_interface.set_string(self._dll.setUpperFrequency, upper_freq_string)
+
+    @property
+    def stop_band_definition(self) -> StopbandDefinition:
+        """Stop band parameter entry option.
+        The default is `RATIO`.
+
+        Returns
+        -------
+        :enum: StopbandDefinition
+        """
+        index = c_int()
+        stop_band_definition = list(StopbandDefinition)
+        status = self._dll.getStopbandDef(byref(index))
+        pyaedt.filtersolutions_core._dll_interface().raise_error(status)
+        stop_band_definition = stop_band_definition[index.value]
+        return stop_band_definition
+
+    @stop_band_definition.setter
+    def stop_band_definition(self, column: StopbandDefinition):
+        status = self._dll.setStopbandDef(column.value)
+        pyaedt.filtersolutions_core._dll_interface().raise_error(status)
+
+    @property
+    def stop_band_ratio(self) -> str:
+        """Filter stop band ratio.
+        The default is `1.2`.
+
+        Returns
+        -------
+        str
+        """
+        stop_band_ratio_string = self._dll_interface.get_string(self._dll.getStopbandRatio)
+        return stop_band_ratio_string
+
+    @stop_band_ratio.setter
+    def stop_band_ratio(self, stop_band_ratio_string):
+        self._dll_interface.set_string(self._dll.setStopbandRatio, stop_band_ratio_string)
+
+    @property
+    def stop_band_frequency(self) -> str:
+        """Filter stop band frequency.
+        The default is `1.2 GHz`.
+
+        Returns
+        -------
+        str
+        """
+        stop_band_frequency_string = self._dll_interface.get_string(self._dll.getStopbandFrequency)
+        return stop_band_frequency_string
+
+    @stop_band_frequency.setter
+    def stop_band_frequency(self, stop_band_frequency_string):
+        self._dll_interface.set_string(self._dll.setStopbandFrequency, stop_band_frequency_string)
+
+    @property
+    def stop_band_attenuation_db(self) -> str:
+        """Filter stop band attenuation in dB.
+        The default is `60 dB`.
+
+        Returns
+        -------
+        str
+        """
+        stop_band_attenuation_db_string = self._dll_interface.get_string(self._dll.getStopbandAttenuationdB)
+        return stop_band_attenuation_db_string
+
+    @stop_band_attenuation_db.setter
+    def stop_band_attenuation_db(self, stop_band_attenuation_db_string):
+        self._dll_interface.set_string(self._dll.setStopbandAttenuationdB, stop_band_attenuation_db_string)
+
+    @property
+    def standard_pass_band_attenuation(self) -> bool:
+        """Filter standard cut status.
```

Comment:
How is the status a Boolean? Usually, for a Boolean property, the description is something like this: Flag indicating if ...." However, I don't know what ``True`` means in regard to the status.